src/js/components/search/visualizations/time/TimeVisualizationSection.jsx

```diff
@@ -49,6 +59,64 @@ export default class TimeVisualizationSection extends React.Component {
         }
     }
 
+    downloadLabel = () => {
```

Comment:
This can just be a ternary  inside the render.

Reply:
I had that originally, but it looked long and awkward to read. But I'll put it back to follow project standards.

Reply:
done